### PR TITLE
add draft mod/comm mgmt lead role

### PR DESCRIPTION
**Pull request by parispittman**
discussion: 
- title 
- location of this file: should it live here or root of this dir: https://github.com/kubernetes/community/tree/master/communication
- what else?

/sig contributor-experience

### CODE REVIEW

**Review by markyjackson-taulia — APPROVED** [review]
/lgtm

**Comment by chris-short** [comment]
I am 100% excited about this role. Brilliant idea and I hope it helps elevate our mentoring program(s).

cc @sheetaljoshi

**Comment by parispittman** [comment]
cc/ @castrojo , @mrbobbytables, @jeefy, @idvoretskyi, @jdumars - what do y'all think?

File: sig-contributor-experience/role-handbooks/comm-mgmt-lead.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+moderation, and administration
+- Keep all moderation guidelines and relevant documentation and up to date
+- Act as liasion between moderation team and Code of Conduct Committee
+
+### [Title] MAY:
+- Delegate the above responsonibilities to the shadow or another moderator 
+- Suggest automation or process improvements to a communication platform
+- Remove another moderators permissions 
+
+### Term:
```
Comment (idvoretskyi) [inline sig-contributor-experience/role-handbooks/comm-mgmt-lead.md:24]: Any limits on terms/re-elections?

Comment (parispittman) [reply sig-contributor-experience/role-handbooks/comm-mgmt-lead.md:24]: im thinking the role is intended to turn over after a year as its necessary to have a shadow; however, if there is no shadow for whatever reason at that time and no suitable replacement, another 1 year? how does that sound?

File: sig-contributor-experience/role-handbooks/comm-mgmt-lead.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- 1 year for bootstrap crew; then on a 6 month cadence for rotation of shadow 
+- Role requires a shadow or to be re-elected 
+
+### How to be a Lead:
+Bootstrapping the role will require two co-leads and an eventual shadow for 
+succession. 
+
+OWNERs for k/community/communition will nominate a lead/successor post bootstrap
```
Comment (jeefy) [inline sig-contributor-experience/role-handbooks/comm-mgmt-lead.md:32]: sp: communition -> communication

File: sig-contributor-experience/role-handbooks/comm-mgmt-lead.md (hunk 1 above)
Comment (jdumars) [reply sig-contributor-experience/role-handbooks/comm-mgmt-lead.md:24]: Unilateral removing of someone's permissions makes me nervous. Can we have a mandatory +1 from another moderator?

**Review by jeefy — CHANGES_REQUESTED** [review]

File: sig-contributor-experience/role-handbooks/comm-mgmt-lead.md (hunk 1 above)
Comment (jeefy) [reply sig-contributor-experience/role-handbooks/comm-mgmt-lead.md:24]: +1 to @jdumars -- removal of elevated permissions should require a +1 from another mod or lead (or shadow?)

**Review by alisondy — APPROVED** [review]

**Pull request closed without merging**